javascript/code-challenges/insertionSort/BLOG.md

```diff
@@ -0,0 +1,24 @@
+# Insertion Sort
+
+## Author: Jeremy Penning
+
+## Pseudo Code
+
+```javascript
+  InsertionSort(int[] arr)
+  
+    FOR i = 1 to arr.length
+    
+      int j <-- i - 1
+      int temp <-- arr[i]
+      
+      WHILE j >= 0 AND temp < arr[j]
+        arr[j + 1] <-- arr[j]
+        j <-- j - 1
+        
+      arr[j + 1] <-- temp
+```
+
+## How it Works
+
+`I need to add back in the images once I get them sliced down`
```

Comment:
I would like to see tests added and when you have a chance to get the visuals up on the README